docs/sharding.md

```diff
@@ -0,0 +1,42 @@
+# Multiple Azure SignalR Service Instances Support
+Currently we support multiple Azure SignalR Service instances **for Persistent mode only**.
+
+## Set up multiple endpoints
+To enable multiple endpoints support, you should: 
+
+1. Set the `ServiceTransportType` to `Persistent` in your configuration.
+```json
+{
+    "AzureSignalRServiceTransportType":"Persistent"
+}
+```
+
+2. Set multiple SignalR connection string entries in your configuration. The keys are in the following format : 
+```
+AzureSignalRConnectionString: <name> : <role>
+```
+You can customize the key prefix `AzureSignalRConnectionString` with `ConnectionStringSetting`. `<name>` is the name of the endpoint and `<role>` is its role (`primary` or `secondary`). Name is optional but it will be useful if you want to further customize the routing behavior among multiple endpoints. If `<role>` is not specified, it will default to `primary`.
```

Comment:
add link of the attribute includes ConnectionStringSetting

Reply:
is primary and secondary case-sensitive? different from the code

Reply:
It's not case-sensitive.